dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/servlet/servicelist/ServiceListGeneratorServlet.java

```diff
@@ -0,0 +1,221 @@
+/**
```

Comment:
We intentionally don't expose this Servlet on liberty, please remove these changes.

Reply:
Changes are reverted.